webapp/utils/polyfill/resizeObserver.js

```diff
@@ -0,0 +1,62 @@
+import { elementOffset } from '../domUtils'
+
+// ResizeObserver polyfill
+export default class ResizeObserver {
+  constructor(callback) {
+    this.observables = []
+    // Array of observed elements that looks like this:
```

Comment:
Shape descrption should be above the definition (if to follow JSDOC style)

Reply:
ok, added the doc as per JSDOC style